src/geo/src/DetectorConstruction.cc

```diff
@@ -17,6 +17,8 @@
 #include <RAT/DetectorFactory.hh>
 #include <RAT/WatchmanDetectorFactory.hh>
 #include <RAT/TheiaDetectorFactory.hh>
+#include <RAT/WatchmanWLSPDetectorFactory.hh>
```

Comment:
File `WatchmanWLSPDetectorFactory.hh` not in this PR

Reply:
Apologies for that, that was from an outdated factory that generated circular plates.  Forgot to remove that line before committing, should be resolved in the most recent commit.